### PR TITLE
task: remove displayName from environments

### PR DESCRIPTION
**Pull request by chriswk**
This will require a change in enterprise as well.

### CODE REVIEW

**Comment by chriswk** [comment]
Actually, since we're using the types we expose for adding the EnvironmentService in Enterprise, this should happen automatically when we update enterprise to use OSS with this task included

**Comment by chriswk** [comment]
We'll need to update the tests to no longer include displayName in Enterprise as well though.

File: src/migrations/20210928065411-remove-displayname-from-environments.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `ALTER TABLE environments
+        DROP COLUMN display_name`,
+        cb,
+    );
+};
+
+exports.down = function (db, cb) {
```
[the remark this answers is not in the document]
Comment (chriswk) [reply src/migrations/20210928065411-remove-displayname-from-environments.js:11]: @sonatype-lift ignore

File: src/migrations/20210928065411-remove-displayname-from-environments.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+'use strict';
```
[the remark this answers is not in the document]
Comment (chriswk) [reply src/migrations/20210928065411-remove-displayname-from-environments.js:1]: @sonatype-lift ignore

**Review by ivarconr — APPROVED** [review]
A lot of simplified tests :P

**Pull request merged**